### ISSUE TITLE
Completion include the ending `\`

### ISSUE THREAD

**Issue by czhang03** [nickcox/cd-extras#13]
include the ending `\` of directories in the completion to simulate the default behavior of PowerShell.

**Comment by nickcox** [comment]
Published with [0.4.0](https://www.powershellgallery.com/packages/cd-extras/0.4.0).

**Comment by thorstenkampe** [comment]
Not able to cd into directory with space in name:
```
> cd cre<TAB>
> cd '.\create database'\
Set-LocationEx : A positional parameter cannot be found that accepts argument '\'.
At C:\Users\thorsten\Documents\PowerShell\Modules\cd-extras\0.4.2\public\Core.ps1:211 char:22
+     setLocation   = {Set-LocationEx @args}
+                      ~~~~~~~~~~~~~~~~~~~~
+ CategoryInfo          : InvalidArgument: (:) [Set-LocationEx], ParameterBindingException
+ FullyQualifiedErrorId : PositionalParameterNotFound,Set-LocationEx
```

**Comment by czhang03** [comment]
swap this line https://github.com/nickcox/cd-extras/blob/master/cd-extras/private/ArgumentCompleter.ps1#L8 and the privious line would fix this issue.

**Comment by nickcox** [comment]
Thanks, guys.

Published as [0.4.3](https://www.powershellgallery.com/packages/cd-extras/0.4.3) and added corresponding [regression test](https://github.com/nickcox/cd-extras/blob/bf7db1d3b824dea3e7f300a6305cf302544f2f43/tests/cd-extras.Tests.ps1#L109) to prevent this from happening again.

**Comment by thorstenkampe** [comment]
After updating to 0.4.3 I get the following error:
```
Register-ArgumentCompleter : Cannot bind argument to parameter 'CommandName' because it is null.
At C:\Users\thorsten\Documents\PowerShell\Modules\cd-extras\0.4.3\private\ArgumentCompleter.ps1:16 char:43
+   Register-ArgumentCompleter -CommandName $commands -ParameterName Pa ...
+                                           ~~~~~~~~~
+ CategoryInfo          : InvalidData: (:) [Register-ArgumentCompleter], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,System.Management.Automation.RegisterArgumentCompleterCommand

Register-ArgumentCompleter : Cannot bind argument to parameter 'CommandName' because it is null.
At C:\Users\thorsten\Documents\PowerShell\Modules\cd-extras\0.4.3\private\ArgumentCompleter.ps1:16 char:43
+   Register-ArgumentCompleter -CommandName $commands -ParameterName Pa ...
+                                           ~~~~~~~~~
+ CategoryInfo          : InvalidData: (:) [Register-ArgumentCompleter], ParameterBindingValidationException
+ FullyQualifiedErrorId : ParameterArgumentValidationErrorNullNotAllowed,System.Management.Automation.RegisterArgumentCompleterCommand
```

My invocation is
```
Import-Module cd-extras -DisableNameChecking
Set-CdExtrasOption -Option NOARG_CD -Value $HOME
```

**Comment by nickcox** [comment]
Eek, sorry about that. A line seems to have [gone missing](https://github.com/nickcox/cd-extras/commit/a49784c7e37a5cef695f584e3a6515ef3fecc283) as part of the previous commit.

Should be fixed with [0.4.4](https://www.powershellgallery.com/packages/cd-extras/0.4.4).
